src/components/common/program-section-templates/single-image-right/ViewSingleImageRight.module.scss

```
@@ -57,6 +57,8 @@ $img-height: 600px;
 .description {
     @include type.body-1-medium;
     word-wrap: break-word;
+    overflow-wrap: break-word;
+    word-break: break-word;
```

Stylelint (17.5.0)

[error] Unexpected deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)